### DISCUSSION TITLE
Welcome to infinite-recharge-robot Discussions!

### DISCUSSION THREAD

**Discussion by cutlasjgh** [wando-advanced-robotics/infinite-recharge-robot#18] [Announcements]
## 👋 Welcome!

Since Discord is not so easy to use for all while on campus, I thought Github discussion might suffice, worth a try.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do with this community.

Some links related to discussions
  * https://docs.github.com/en/github/site-policy/github-community-guidelines
  * https://docs.github.com/en/discussions/guides/best-practices-for-community-conversations-on-github
  * https://docs.github.com/en/organizations/collaborating-with-your-team/about-team-discussions
  * https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository

   Remember You may wish to respond to something by disagreeing with it. That’s fine. But remember to criticize ideas, not people.  Respect each other, be polite please and  no name-calling of course,  other rules are here https://docs.github.com/en/github/site-policy/github-community-guidelines .

  I believe we'll have to start a new discussion for the new 2022 Robot build also just FYI.

**Comment by abrightwell** [comment]
Adam - Mentor - tag me on any conversation, pull request, issue, commit, etc that you my input - @abrightwell. Since I work with GH daily, this is a great way to get my attention and ask me questions outside of team meetings.  Also, feel free to send me an email at adam.brightwell@gmail.com. I can do video chats as needed for times that I am not able to make it to the school.
